tests/unit/source_code/test_notebook.py

```diff
@@ -215,7 +215,7 @@ def test_notebook_builds_python_dependency_graph(mock_path_lookup):
     assert graph.all_paths == {mock_path_lookup.cwd / path for path in paths}
 
 
-def test_detects_manual_migration_in_dbutils_notebook_run_in_python_code_():
+def test_detects_manual_migration_in_dbutils_notebook_run_in_python_code_() -> None:
```

Comment:
Why add the type hint only to these tests?

Reply:
In this case the situation is:

 - Mypy doesn't lint function bodies if the function definition doesn't include any type hints.
 - The other tests all have a type hint in them somewhere (normally the arguments).
 - These are no-arg tests, so unless we hint the return type they're untyped and therefore Mypy ignores them. (This is what those warnings in the PR description are all about.)

Arguably we should be fully hinting everything else as well, but it wasn't the focus of this PR which is just to ensure that most function bodies are being checked and remove the warnings during `make verify`.

Reply:
… and I just noticed there are other functions in the same file without any type hints. I'm not sure why they weren't triggering the same warnings. 🤔